app/build.gradle

```diff
@@ -60,6 +64,7 @@ dependencies {
 
     //Dagger Hilt
     implementation "com.google.dagger:hilt-android:$versions.hilt"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
Can you please check, on this dependency. I believe we don't require this and it is safe to remove it.